docs/StyleCustomization.md

```diff
@@ -0,0 +1,67 @@
+# Style Customization
+
+By default Hydrogen leans towards plain styling so that its elements can be customized easily to your liking.
+
+Below are some tips to get started with styling Hydrogen. If you are unfamiliar with this topic there is more detailed information availible in the [Atom Flight Manual](http://flight-manual.atom.io/using-atom/sections/basic-customization/#style-tweaks).
+
+Start out by opening your styles.less file or searching your command palette for "stylesheet", you will see atom's: **Application: Open Your Stylesheet** command.
+
+To follow these examples make sure you have theme variables imported by puting this at the top of your styles.less file.
+
+```less
+@import "ui-variables";
+@import "syntax-variables";
+
+```
+
+In the example below there are color variables like `@my-color-variable` used. Of course, you can change these as desired.
+
+## Style Examples
+### Multiline Result View
+
+Change the result view background color for multi-line output:
+
+In your styles.less:
+
+```less
+.hydrogen {
+  .multiline-container {
+    .cell_display {
+      background-color: @background-color-highlight;
+    }
+  }
+}
+```
+
+
+Change the icon color when you hover over the toolbar icons:
+
+```less
+.hydrogen {
+  .multiline-container {
+    .toolbar .icon:hover {
+      background-color: @background-color-selected;
+    }
+  }
+}
+```
+
+### Importing Hydrogen Style Variables
+
+If you want to use Hydrogen's style variables you can do so by adding this to your styles.less:
+
+```less
+  @import "./packages/Hydrogen/styles/hydrogen.less";
+```
+
+Once you've done this you will have access to any style variables within the Hydrogen package:
+
+```less
+@hydrogen-light-background
```

Comment:
> @hydrogen-light-background: red, will Hydrogen recognize that?

Technically not Hydrogen doing the recognizing :smirk:, but yes this works!

You can do crazy stuff with less, but it is a little hard to read and hackish. Some of the nesting and grouping property values is really good, but variables are wierd. 

To do what I do in these examples you can even go:
```less
@packagename: ~'.hydrogen';
@multiline-container: ~'@{packagename} .multiline-container';
@cell_display: ~'@{multiline-container} .cell_display';


@{multiline-container} .toolbar .icon:hover {
  background-color: @background-color-selected;
}

@{cell_display} {
  background-color: @background-color-highlight;
}
```